### PR TITLE
Fix authentication for kotlin by using login.salesforce.com

### PR DESCRIPTION
**Pull request by rileymacdonald-sf**

### CODE REVIEW

**Comment by rileymacdonald-sf** [comment]
@sfyz Please see documentation changes in `README.md`.

File: Examples/SnapinsSDKKotlinExample/README.md [hunk 1]
```diff
@@ -21,9 +21,13 @@ Update the required string configurations in [strings.xml](app/src/main/res/valu
 valid data to ensure the Snap-ins SDK will function as expected.
 
 
-### Using Authenticated Users
+### Using Authentication
+
+In order to use authentication, developers can:
+* Provide a valid domain in [servers.xml](app/src/main/res/xml/servers.xml).
+* Use the provided default <a href="https://login.salesforce.com">login.salesforce.com</a> domain
```
Comment (sfyz) [inline Examples/SnapinsSDKKotlinExample/README.md]: Wonder if you should make make it clear that this is an either/or list of things... Maybe just say: "developer can either:" before the items...?

Comment (rileymacdonald-sf) [reply Examples/SnapinsSDKKotlinExample/README.md]: Sure, works for me.

**Comment by rileymacdonald-sf** [comment]
This pull request is ready to be merged.

**Pull request merged**